Dockerfile

```diff
@@ -0,0 +1,26 @@
+FROM ghcr.io/linuxserver/baseimage-alpine:edge AS buildstage
+
+ARG BUILD_DATE
+ARG VERSION
+LABEL build_version="Linuxserver.io version:- ${VERSION} Build-date:- ${BUILD_DATE}"
+LABEL maintainer="Quietsy"
+
+COPY root/app/requirements.txt /tmp/requirements.txt
```

Comment:
Unless there's a specific reason not to you can just move the whole `COPY root/ /` up here.

Reply:
It speeds up the dev cycle locally not having to rebuild from this point when changing the code